run-build

```diff
@@ -88,6 +88,11 @@ class ImageBuildRoot(object):
         keyring = eib.get_keyring(config)
 
         # Wipe the apt cache directory if it's grown too large.
+        #
+        # FIXME: This is currently doing nothing because mmdebstrap
+        # throws an error if you seed /var/cache/apt/archives in the
+        # root before it runs. To make this work something like
+        # apt-cacher is needed to handle it it the network level.
```

Comment:
Shouldn't we remove this block of dead code then? IIUC this will never be pre-seeded if we cache in the network level, apt will still download the packages even if from a server on the same host or local network.

Reply:
I think cache at the network level was poorly worded. It would be handled by proxying the HTTP requests and returning `.deb` files from disk or saving them when they're new. Basically a little local CDN.

That said, I don't think I'll be working on it any time soon. Maybe comment out the code for now?

Reply:
Yes, that is what I had in mind. I guess the question is, would we still want to go and clean the proxy cache directory in that case right from eos-image-builder? I thought most proxies would control that themselves.

Reply:
I suppose they could, but I think eos-image-builder would also be the one starting and stopping the proxy. In that case, the brute force cleanup would still be reasonable to have.

However, since I don't really know what it would look like, I'll just comment it out for now.